Ports/freetype/patches/0001-libtool-Enable-shared-library-support-for-SerenityOS.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Daniel Bertalan <dani@danielbertalan.dev>
+Date: Tue, 1 Aug 2023 20:34:42 +0200
```

Comment:
This looks like author and date have been reset. I don't particularly mind it, but was that a conscious choice or did something happen while using `./package.sh dev`?

Reply:
I had to fix up some things, so I ended up committing manually which used my user ID. Will fix up the patch.